cropro.py

```diff
@@ -179,6 +201,22 @@ def initUI(self):
 
         self.setWindowTitle('Cross Profile Search and Import')
         self.exec_()
+    
+    def playAudio(self):
+        if not self.audioCheckBox.isChecked():
+            return
+        # get the note ids of all selected notes
+        noteIds = [self.noteListModel.itemFromIndex(idx).data() for idx in self.noteListView.selectedIndexes()]
+        firstCardId = noteIds[0]
+        firstCard = self.otherProfileCollection.getNote(firstCardId)
+        audioPath = ''
+        for field in firstCard.fields:
```

Comment:
This is just the most basic find operation. could be replaced with a nice and pretty single line